### PR TITLE
Prepare ledger release

### PR DESCRIPTION
**Pull request by lehins**
# Description

This PR includes various fixups needed for the next release to CHaPs, as well as the follow up task of updating Changelogs

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [x] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
      [versioning process](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process).
- [x] The version bounds in `.cabal` files for all affected packages are updated. **If you change the bounds in a cabal file, that package itself must have a version increase.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process))
- [x] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages. **New section is never added with the code changes.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd))
- [x] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
- [x] Cabal files are formatted (use `scripts/cabal-format.sh`)
- [x] [`hie.yaml`](https://github.com/intersectmbo/cardano-ledger/blob/master/hie.yaml) has been updated (use `scripts/gen-hie.sh`)
- [x] Self-reviewed the diff

### CODE REVIEW

File: libs/cardano-protocol-tpraos/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Version history for `cardano-protocol-tpraos`
 
-## 1.0.3.8
+## 1.1.0.0
 
-*
+* Change teh type of `bsize` and `hBbsize` to `Word32`
```
Comment (teodanciu) [inline libs/cardano-protocol-tpraos/CHANGELOG.md]: Small typo here. We can fix it later, probably not worth another CI run

Comment (lehins) [reply libs/cardano-protocol-tpraos/CHANGELOG.md]: Yeah, I noticed that too. I'll fix that at some later point. I already have a release ready for this: https://github.com/IntersectMBO/cardano-haskell-packages/pull/645

File: eras/babbage/test-suite/CHANGELOG.md
Comment (teodanciu) [inline eras/babbage/test-suite/CHANGELOG.md:1]: Have we missed the changelog entry update  for babbage in `eras/babbage/impl/CHANGELOG.md`  ?

**Review by teodanciu — APPROVED** [review]
Looks good to me, great that the CHaP build is passing.

File: eras/babbage/test-suite/CHANGELOG.md
Comment (lehins) [reply eras/babbage/test-suite/CHANGELOG.md:1]: No we didn't. Version for `cardano-ledger-babbage` was wrong, so I had to fix that, which resulted in not having to update the changelog

**Comment by lehins** [comment]
Looks like CI caught a bug at the very last minute. I'll have to redo the release once we fix it.
CC. @aniketd

**Review by Soupstraw — APPROVED** [review]
LGTM

File: libs/cardano-protocol-tpraos/CHANGELOG.md (hunk 1 above)
Comment (lehins) [reply libs/cardano-protocol-tpraos/CHANGELOG.md]: Fixed.

**Comment by lehins** [comment]
> Looks like CI caught a bug at the very last minute. I'll have to redo the release once we fix it.

I decided to make a release regardless of this bug. We'll make a patch release for cardano-ledger-conway, once we squash it.

**Pull request merged**